vendors/microchip/secure_elements/app/example_trust_chain_tool/README.md

```diff
@@ -90,17 +125,22 @@ xxd -r -ps DevicePublicKeyAsciiHex.txt DevicePublicKeyDer.bin
 openssl ec -inform der -in DevicePublicKeyDer.bin -pubin -pubout -outform pem -out public_key.pem
 ```
 
-6) Don't forget to disable the temporary key generation setting you enabled above. Otherwise, the device will create yet another key pair, and you will have to repeat the previous steps:
+6) Don't forget to disable the temporary key generation setting you enabled above. 
+Otherwise, the device will create yet another key pair, and you will have to repeat the previous steps:
 
 ```
 #define keyprovisioningFORCE_GENERATE_NEW_KEY_PAIR 0
 ```
-7) Run the ca_create_device script, using the 18 characters following CN= for the --sn input:
+7) Create your device certificate using the ca_create_device script. 
+Use the 18 characters following CN= for the --sn input, and the public_key.pem file
+that you formatted step 5 as your public key.
 
 ```
-> ca_create_device.py  --sn 01234D2C14CBEAD5EE --cert signer-ca.crt --key signer-ca.key --file public_key.pem
+>python ca_create_device.py  --sn 01234D2C14CBEAD5EE --cert signer-ca.crt --key signer-ca.key --file public_key.pem
 ```
 
+You now have a device certificate called device.crt.
+
```

Comment:
I think we could delete the next two paragraphs.

Reply:
yes